lib/kube/proxy/forwarder.go

```diff
@@ -307,6 +308,13 @@ type authContext struct {
 	sessionTTL time.Duration
 }
 
+type endpoint struct {
+	// targetAddr is a direct network address.
+	addr string
+	// serverID is an address reachable over a reverse tunnel.
```

Comment:
Is it an ID or an address?

Reply:
Yes... a bit of both. Updated comment to clarify here and from original comment in `teleportClusterClient.serverID`.